sdks/python/apache_beam/transforms/userstate.py

```diff
@@ -305,23 +323,37 @@ def validate_stateful_dofn(dofn):
                        (timer_spec, method_name, dofn))
 
 
-class RuntimeTimer(object):
+class BaseTimer(object):
```

Comment:
I added a base class so that I could have a return type for `UserStateContext.get_timer()`.  Also see `bundle_processor. OutputTimer` (at the top of the PR).  

An alternative to this would be to create a `Protocol`, but that seemed needlessly complex.

Reply:
This is good.